packages/adapter-static/index.d.ts

```diff
@@ -1,7 +1,7 @@
-declare function plugin(options: {
+declare function plugin(options?: {
 	pages?: string;
 	assets?: string;
-	fallback?: string;
+	fallback?: string | null;
```

Comment:
I wonder if we should make the default `undefined` to simplify the types?

Reply:
From what I can tell that'd be fine, I can't see any places where fallback couldn't be `undefined` instead of `null`, and lots of the other adapters don't specify a `fallback` key in `utils.prerender`, so that seems pretty safe. Will push that change now!